tests/wormhole/test_silicon_driver_wh.cpp

```diff
@@ -981,6 +981,11 @@ TEST(SiliconDriverWH, LargeAddressTlb) {
     // Read the scratch register via a different TLB, different code path:
     cluster.read_from_device(&value2, ARC_CORE, addr, sizeof(uint32_t), "REG_TLB");
 
+    // Mask off lower 16 bits; FW changes these dynamically:
```

Comment:
Could it be the same cause as for https://github.com/tenstorrent/tt-umd/issues/362 , since that one also has lower 16 bits changing?

Reply:
Don't think so - this register is controlled by ARC FW.
That problem looks like Tensix reset in a weird state.